packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -789,6 +789,39 @@ Widget inputDecorationSuffixButton(IconData icon, VoidCallback onPressed) {
   );
 }
 
+class OutlinedRowGroup extends StatelessWidget {
+  const OutlinedRowGroup({@required this.children, this.borderColor});
+
+  final List<Widget> children;
+
+  final Color borderColor;
+
+  @override
+  Widget build(BuildContext context) {
+    final color = borderColor ?? Theme.of(context).focusColor;
+    final childrenWithOutlines = <Widget>[];
+    for (int i = 0; i < children.length; i++) {
+      childrenWithOutlines.addAll([
+        Container(
+          decoration: BoxDecoration(
+            border: Border(
+              left: i == 0 ? BorderSide(color: color) : BorderSide.none,
```

Comment:
why is only the left side special cased and not the right side.?

Reply:
We do this to make sure there are not any double borders. The right side of the previous element will serve as the left border for the next element (for all except the first since the first does not have a previous element)